src/FluentValidation/Resources/Languages/FrenchLanguage.cs

```diff
@@ -46,6 +46,7 @@ internal class FrenchLanguage : Language {
 			Translate<EmptyValidator>("'{PropertyName}' devrait être vide.");
 			Translate<NullValidator>("'{PropertyName}' devrait être vide.");
 			Translate<EnumValidator>("'{PropertyName}' a une plage de valeurs qui n'inclut pas '{PropertyValue}'.");
+			Translate<StringEnumValidator>("'{PropertyName}' a des membres qui n'incluent pas '{PropertyValue}'.");

```

Comment:
Let's remove this message and use the `EnumValidator` message instead, so re-translation isn't necessary.